src/data/content/community/notizie/20230314-nome-in-codice-italia.yaml

```diff
@@ -0,0 +1,163 @@
+        addonClasses: mt-3 text-uppercase
+    # pretext:
+    #   icon:
+    #     icon: sprites.svg#it-info-circle
+    #     size: sm
+    #   text: In breve
+    # text: |
+    #   Lorem ipsum
+    share:
+      btnId: shareButton
+      button:
+        label: Condividi
+        addonStyle: btn-share-hero btn-dropdown text-primary mb-5 mt-lg-4 ms-auto
+        icon:
+          icon: sprites.svg#it-more-items
+          size: sm
+          color: primary
+          addonClasses: ms-3
+      list:
+        isShare: true
+
+    kangaroo:
+      id: kangarooExample
+      titleSr: Metadati e link per approfondire
+      noPadding: true
+      tagsLabel: Parliamo di
+      color: secondary
+      icon:
+        icon: sprites.svg#it-horn
+        size: sm
+        color: secondary
+      chips:
+        - label: Design system
+          url: "#"
+          color: primary
+          size: lg
+        - label: Community
+          url: "#"
+          color: primary
+          size: lg
+      personalInfo:
+         items:
+          - title: Tempo di lettura
+            icon:
+              icon: "sprites.svg#it-clock"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: 2 minuti
+          - title: Data
+            icon:
+              icon: "sprites.svg#it-calendar"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: 14 marzo 2023
+
+  imageIcons:
+    image: /images/community/20230314-nome-in-codice-italia-hero.jpg
+    background: light
+    alt: ""
+    
+
+  sectionsEditorial :
+    - menu: false
+      centered: true
+      # title: Progettare interfacce è un atto di responsabilità, significa decidere continuamente chi includere, o escludere
+      # text: 
+      id: corpoNotizia
+      components:
+        - name: TextImageCta
+          # title: Titoletto interno
+          text: |
+            **Designers Italia lancia un sondaggio per definire il nome del design system**, una delle risorse fondamentali per realizzare progetti digitali della Pubblica Amministrazione di qualità, accessibili, e inclusivi. La consultazione informale sarà aperta per un mese su Forum Italia, fino al **14 aprile.**
+
+            ## Cos’è il design system
+            Il [design system](/design-system/) può essere visto come una **“scatola dei mattoncini”** che contiene, al suo interno, una serie di librerie di componenti pronti all'uso, principi e guide pratiche per progettare interfacce digitali, insieme a una raccolta di soluzioni pronte all’uso (design pattern) e casi studio. Una serie di strumenti a disposizione di chi progetta e sviluppa interfacce digitali che permette a PA, in-house e fornitori di **progettare esperienze semplici e che funzionano**, risparmiare tempo e risorse, oltre che di usufruire di materiali open source costantemente aggiornati, documentazione tecnica, esempi e buone pratiche di versionamento. 
```

Comment:
@gianvitofanelli double "pronti all'uso" for "componenti" and "pattern" on the same phrase... is it correct?